docs/docs/streamr-network/incentives/network-incentives.md

```diff
@@ -30,6 +30,109 @@ The introduction of token economics defines three new roles: [Sponsor](../networ
 
 It should be noted that the roles are independent of each other and can be mixed and matched by all actors depending on their goals, for example, the same person could be a Sponsor, Publisher, and a Delegator.
 
+## Technical description
+
+The tokenomics are mainly governed by the following two smart contracts: the Operator contract and the Sponsorship contract. The Operator contract represents an [operator running a Streamr node in the Streamr network](../network-roles/operators.md). The Sponsorship contract holds [the sponsors'](../network-roles/sponsors.md) DATA tokens that `sponsor` a stream, and allocates them as earnings to the stakers.
+
+Stakers are Operator contracts controlled by their operators. The DATA tokens held by the Operator contract are the funds that the operator can `stake` into Sponsorship contracts. By staking into Sponsorship, the operator commits to servicing the sponsored stream.
```

Comment:
`Stakers` is newly introduced term in the docs, on the one hand its easy to understand, on the other it adds another term into the mix. IIUC A `staker` is synonymous with the `Operator` and the `Operator contract`. Should we stick to `Operators` rather than `stakers` @jtakalai?

See the original terminology breakdown here:
https://docs.google.com/spreadsheets/d/1YBbqo__5B47-9flGYkOVRonU6cChVthLZhH8ldj47lo/edit?gid=0#gid=0